tnef.go

```diff
@@ -73,6 +73,14 @@ type Data struct {
 	Attributes  []MAPIAttribute
 }
 
+// The file did not have a valid TNEF signature,
+// so there is no chance that it was TNEF format.
+var ErrInvalidSignature = errors.New("signature didn't match valid TNEF file")
```

Comment:
I think the comment would read a bit clearer as:

```
// ErrNoMarker signals that the file did not start with the fixed TNEF marker,
// meaning it's not in the TNEF file format we recognize (e.g. it just has the
// .tnef extension, or a wrong MIME type).
var ErrNoMarker = errors.New("file did not begin with a TNEF marker")
```

"signature" is a bit ambiguous; I initially thought it's a signing signature for security (it's not).